site/components/svg-previewer/svg-previewer.tsx

```diff
@@ -155,35 +161,72 @@ export const SvgPreviewer: React.FC = () => {
 
   return (
     <StyledSvgPreviewerContainer>
-      <StyledButtonsContainer>
-        <ThemeControls
-          setSvgCodeGroup={setSvgCodeGroup}
-          hexesObj={hexesObj}
-          baseSvgCodeGroup={baseSvgCodeGroup}
-          svgCodeGroup={svgCodeGroup}
-          getThemeFromList={getThemeFromList}
-          setIds={setIds}
-          currentThemeName={currentThemeName}
-          setCurrentThemeName={setCurrentThemeName}
-        />
-
-        <DownloadControls
-          hexesObj={hexesObj}
-          baseSvgCodeGroup={baseSvgCodeGroup}
-          svgCodeGroup={svgCodeGroup}
-        />
+      <StyledButtonsContainer
+        rows={{xs: 'sizing080'}}
+        alignItems="center"
+        justifyContent="sapce-between"
+        columns="1fr 1fr"
+        overrides={{
+          paddingInline: '10%',
+          paddingBlock: '8px',
+          minWidth: '100%',
+        }}
+      >
+        <GridLayout
+          columns="1fr 1fr"
+          alignItems="center"
+          rows={{xs: 'sizing080'}}
+        >
+          <ThemeControls
+            setSvgCodeGroup={setSvgCodeGroup}
+            hexesObj={hexesObj}
+            baseSvgCodeGroup={baseSvgCodeGroup}
+            svgCodeGroup={svgCodeGroup}
+            getThemeFromList={getThemeFromList}
+            setIds={setIds}
+            currentThemeName={currentThemeName}
+            setCurrentThemeName={setCurrentThemeName}
+          />
+        </GridLayout>
+        <GridLayout
+          columns="1fr 1fr 1fr"
+          alignItems="baseline"
+          rows={{xs: 'sizing080'}}
+        >
+          <DownloadControls
+            hexesObj={hexesObj}
+            baseSvgCodeGroup={baseSvgCodeGroup}
+            svgCodeGroup={svgCodeGroup}
+          />
+        </GridLayout>
       </StyledButtonsContainer>
 
       {svgCodeGroup && baseSvgCodeGroup && (
-        <StyledSvgGroupContainer>
-          {svgCodeGroup.map((svgCode, index) => (
-            <StyledSingleSvgWrapper key={baseSvgCodeGroup[index].name}>
-              <P>{baseSvgCodeGroup[index].name}</P>
-              {/* eslint-disable-next-line react/no-danger */}
-              <div dangerouslySetInnerHTML={{__html: sanitizer(svgCode)}} />
-            </StyledSingleSvgWrapper>
-          ))}
-        </StyledSvgGroupContainer>
+        <Scroll
+          id="scroll"
+          snapAlign="center"
+          controls="static"
+          stepDistance={window.innerWidth}
+        >
+          <GridLayout
+            autoRows="1fr"
+            autoFlow="column dense"
+            overrides={{
+              marginBlock: '10vh',
+              maxHeight: '80vh',
+            }}
+          >
+            {svgCodeGroup.map((svgCode, index) => (
+              <ScrollSnapAlignment key={getSSRId()}>
```

Comment:
I think is better not to have key rather than use this function like this.
better use the `key={baseSvgCodeGroup[index].name}` as code bellow or something like this
something like that might work better
```suggestion
              <ScrollSnapAlignment key={svgCode.substr(0,100)}>
```